src/protocols/multi_party_ecdsa/gg_2020/party_i.rs

```diff
@@ -484,13 +490,13 @@ impl SignKeys {
     }
 
     pub fn create(
-        private: &PartyPrivate,
+        private_x_i: &FE,
```

Comment:
is this change mandatory? 
can't we remove `u_i` from PartyPrivate instead?

Reply:
`u_i` is being used in every method of PartyPrivate, so we will have to add `u_i` as a new argument to every method. Is it worth it?

Reply:
Here is what I suggest: 
- If I recall correctly , PartyPrivate methods are mostly for 2 party refresh and recovery purposes. Since we are introducing new protocol for refresh, we will probably be able to remove/replace few of the PartyPrivate methods. I think that we can save this work to later time. 
- What is relevant to our case is that we use u_i for `SignKeys` although you just showed we in this PR that this is unnecessary. My end goal here is to be able to run threshold signing without `u_i` (meaning only with linear key). This is because I want to refresh only linear keys and not keep the additive keys after key generation. Do you think we can change SignKeys to be created without `u_i` ?